e3sm_to_cmip/cmor_handlers/mpas_vars/hfsifrazil.py

```diff
@@ -74,28 +79,39 @@ def handle(infiles, tables, user_input_path, **kwargs):
     ds = mpas.add_depth(ds, dsMesh)
     ds.compute()
 
-    ds = mpas.remap(ds, 'mpasocean', mappingFileName)
+    ds = mpas.remap(ds, "mpasocean", mappingFileName)
 
-    util.setup_cmor(var_name=VAR_NAME, table_path=tables, table_name=TABLE,
-            user_input_path=user_input_path)
+    util.setup_cmor(
+        var_name=VAR_NAME,
+        table_path=tables,
+        table_name=TABLE,
+        user_input_path=user_input_path,
+    )
 
     # create axes
-    axes = [{'table_entry': 'time',
-             'units': ds.time.units},
-            {'table_entry': 'depth_coord',
-             'units': 'm',
-             'coord_vals': ds.depth.values,
-             'cell_bounds': ds.depth_bnds.values},
-            {'table_entry': 'latitude',
-             'units': 'degrees_north',
-             'coord_vals': ds.lat.values,
-             'cell_bounds': ds.lat_bnds.values},
-            {'table_entry': 'longitude',
-             'units': 'degrees_east',
-             'coord_vals': ds.lon.values,
-             'cell_bounds': ds.lon_bnds.values}]
+    axes = [
+        {"table_entry": "time", "units": ds.time.units},
+        {
+            "table_entry": "depth_coord",
+            "units": "m",
+            "coord_vals": ds.depth.values,
+            "cell_bounds": ds.depth_bnds.values,
+        },
+        {
+            "table_entry": "latitude",
+            "units": "degrees_north",
+            "coord_vals": ds.lat.values,
+            "cell_bounds": ds.lat_bnds.values,
+        },
+        {
+            "table_entry": "longitude",
+            "units": "degrees_east",
+            "coord_vals": ds.lon.values,
+            "cell_bounds": ds.lon_bnds.values,
+        },
+    ]
     try:
-        mpas.write_cmor(axes, ds, VAR_NAME, VAR_UNITS, positive='down')
+        mpas.write_cmor(axes, ds, VAR_NAME, VAR_UNITS)
```

Comment:
This is the only "real" change here.  The rest is linting (and in its own commit).